.eslintrc.json

```diff
@@ -46,7 +46,13 @@
     "@typescript-eslint/no-var-requires": 0,
     "@typescript-eslint/explicit-function-return-type": 0,
     "@typescript-eslint/explicit-module-boundary-types": "off",
-    "no-underscore-dangle": "off"
+    "no-underscore-dangle": "off",
+    "indent": [2, 4, {"SwitchCase": 1}],
+    "key-spacing": [2, {
+      "singleLine": {
+        "beforeColon": false,
+        "afterColon": true
+      }}]
```

Comment:
what does this do?

Reply:
It ensures uniform spacing around semicolons.  For example: 

`{ "foo" :  42 }; is converted to { "foo":  42 };`